src/config/getDomainSpecificContent.js

```diff
@@ -0,0 +1,33 @@
+const { ACCOUNT_ID } = require('../api.v2/constants');
+
+const domainSpecificContent = {
+  test: {
+    githubOrganisationName: 'org',
+    adminEmail: 'admin@example.com',
+    enforcePrivacyPolicyAgreement: true,
```

Comment:
As it is, local development is broken for me (I log in to HMS staging with no privacy policy)

```suggestion
    enforcePrivacyPolicyAgreement: false,
```

Reply:
This is for the test environment, did you run the api with `DEV_ACCOUNT=HMS npm start` ?